taichi/profiler.cpp

```diff
@@ -139,6 +139,7 @@ class CPUProfiler : public ProfilerBase {
 
 std::unique_ptr<ProfilerBase> make_profiler(Arch arch) {
   if (arch == Arch::x86_64 || arch == Arch::arm) {
+    // TODO: Arch::metal also uses CPUProfiler, rename it to DefaultProfiler?
```

Comment:
Good idea!

Reply:
[master c5e0da63] CPUProfiler->DefaultProfiler; make_profiler now works for Arch::metal

Reply:
Nice!